app.py

```diff
@@ -93,8 +94,11 @@ def get_iss(altitude, latitude, longitude):
                 append_sat_keys(actual_iss, beacon)
             break
 
-    return actual_iss
+    return actual_iss or ""
```

Comment:
The app was coded to handle a null value being returned for the iss, so I'm not sure if this change will break it